app/app.go

```diff
@@ -844,6 +848,17 @@ func (app *UmeeApp) RegisterAPIRoutes(apiSvr *api.Server, apiConfig config.APICo
 	// register legacy and grpc-gateway routes for all modules
 	ModuleBasics.RegisterRESTRoutes(clientCtx, apiSvr.Router)
 	ModuleBasics.RegisterGRPCGatewayRoutes(clientCtx, apiSvr.GRPCGatewayRouter)
+
+	if apiConfig.Swagger {
+		RegisterSwaggerAPI(apiSvr.ClientCtx, apiSvr.Router)
+	}
+}
+
+// RegisterSwaggerAPI registers swagger route with API Server
+func RegisterSwaggerAPI(_ client.Context, rtr *mux.Router) {
+	// register app's OpenAPI routes.
+	rtr.Handle("/swagger.yaml", http.FileServer(http.FS(swagger.Docs)))
+	rtr.HandleFunc("/swagger/", openapiconsole.Handler(Name, "/swagger.yaml"))
```

Comment:
Move this into `swagger.go` please 😄

Reply:
done at https://github.com/umee-network/umee/pull/1203/commits/d1f7c30ca273b1a4ce490a76a394a7e812df959a